Column name casing in DISTINCT ON statements can throw error

When I write the following DISTINCT ON statement:
```
Merchant := DISTINCT Merchant ON merchantid ORDER BY updatedTime DESC;
```
The compiler throws an exception:
```
Caused by: org.apache.calcite.runtime.CalciteContextException: At line 0, column 0: Column 'merchantid' is ambiguous
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
	at org.apache.calcite.runtime.Resources$ExInstWithCause.ex(Resources.java:467)
	at org.apache.calcite.sql.SqlUtil.newContextException(SqlUtil.java:915)
	at org.apache.calcite.sql.SqlUtil.newContextException(SqlUtil.java:900)
	at org.apache.calcite.sql.validate.SqlValidatorImpl.newValidationError(SqlValidatorImpl.java:4881)
	at org.apache.calcite.sql.validate.DelegatingScope.fullyQualify(DelegatingScope.java:496)
	at org.apache.calcite.sql.validate.SqlValidatorImpl.findTableColumnPair(SqlValidatorImpl.java:3492)
	at org.apache.calcite.sql.validate.SqlValidatorImpl.isRolledUpColumn(SqlValidatorImpl.java:3524)
	at org.apache.calcite.sql.validate.SqlValidatorImpl.expandStar(SqlValidatorImpl.java:597)
	at org.apache.calcite.sql.validate.SqlValidatorImpl.expandSelectItem(SqlValidatorImpl.java:392)
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateSelectList(SqlValidatorImpl.java:4075)
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateSelect(SqlValidatorImpl.java:3361)
	at org.apache.calcite.sql.validate.SelectNamespace.validateImpl(SelectNamespace.java:60)
	at org.apache.calcite.sql.validate.AbstractNamespace.validate(AbstractNamespace.java:84)
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateNamespace(SqlValidatorImpl.java:998)
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateQuery(SqlValidatorImpl.java:976)
	at org.apache.calcite.sql.SqlSelect.validate(SqlSelect.java:232)
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validateScopedExpression(SqlValidatorImpl.java:953)
	at org.apache.calcite.sql.validate.SqlValidatorImpl.validate(SqlValidatorImpl.java:705)
	at com.datasqrl.calcite.QueryPlanner.planRoot(QueryPlanner.java:179)
	at com.datasqrl.calcite.QueryPlanner.planCalcite(QueryPlanner.java:175)
	at com.datasqrl.calcite.QueryPlanner.planCalcite(QueryPlanner.java:169)
	at com.datasqrl.calcite.QueryPlanner.plan(QueryPlanner.java:156)
	at com.datasqrl.plan.validate.ScriptPlanner.postvisit(ScriptPlanner.java:372)
	at com.datasqrl.plan.validate.ScriptPlanner.visit(ScriptPlanner.java:868)
	at com.datasqrl.plan.validate.ScriptPlanner.visit(ScriptPlanner.java:108)
	at com.datasqrl.calcite.visitor.SqlNodeVisitor.accept(SqlNodeVisitor.java:48)
	at com.datasqrl.plan.validate.ScriptPlanner.validateStatement(ScriptPlanner.java:999)
	at com.datasqrl.plan.ScriptPlanner.plan(ScriptPlanner.java:56)
	... 85 more
Caused by: org.apache.calcite.sql.validate.SqlValidatorException: Column 'merchantid' is ambiguous
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
	at org.apache.calcite.runtime.Resources$ExInstWithCause.ex(Resources.java:467)
	at org.apache.calcite.runtime.Resources$ExInst.ex(Resources.java:560)
	... 112 mor
```
If I change the name of the column to `merchantId` (which is the name in the schema) then it works.

--- a/sqrl-calcite/src/main/java/com/datasqrl/calcite/SqrlToSql.java
+++ b/sqrl-calcite/src/main/java/com/datasqrl/calcite/SqrlToSql.java
@@ -1,15 +1,14 @@
 package com.datasqrl.calcite;
 
-import static com.datasqrl.plan.validate.ScriptPlanner.addError;
 import static com.datasqrl.plan.validate.ScriptPlanner.isSelfTable;
 import static com.datasqrl.plan.validate.ScriptPlanner.isVariable;
 import static org.apache.calcite.sql.SqlUtil.stripAs;
 
-import com.datasqrl.calcite.SqrlToSql.Context;
-import com.datasqrl.calcite.SqrlToSql.Result;
 import com.datasqrl.calcite.NormalizeTablePath.PathItem;
 import com.datasqrl.calcite.NormalizeTablePath.SelfTablePathItem;
 import com.datasqrl.calcite.NormalizeTablePath.TablePathResult;
+import com.datasqrl.calcite.SqrlToSql.Context;
+import com.datasqrl.calcite.SqrlToSql.Result;
 import com.datasqrl.calcite.function.SqrlTableMacro;
 import com.datasqrl.calcite.schema.sql.SqlBuilders.SqlAliasCallBuilder;
 import com.datasqrl.calcite.schema.sql.SqlBuilders.SqlCallBuilder;
@@ -23,7 +22,6 @@ import com.datasqrl.canonicalizer.NamePath;
 import com.datasqrl.canonicalizer.ReservedName;
 import com.datasqrl.plan.hints.TopNHint.Type;
 import com.datasqrl.util.SqlNameUtil;
-import com.fasterxml.jackson.databind.jsontype.BasicPolymorphicTypeValidator.NameMatcher;
 import com.google.common.base.Preconditions;
 import com.google.common.collect.ArrayListMultimap;
 import com.google.common.collect.Multimap;
@@ -47,7 +45,6 @@ import org.apache.calcite.schema.Function;
 import org.apache.calcite.schema.FunctionParameter;
 import org.apache.calcite.schema.TableFunction;
 import org.apache.calcite.sql.CalciteFixes;
-import org.apache.calcite.sql.SqlBasicCall;
 import org.apache.calcite.sql.SqlCall;
 import org.apache.calcite.sql.SqlFunctionCategory;
 import org.apache.calcite.sql.SqlHint;

--- a/sqrl-calcite/src/main/java/com/datasqrl/calcite/SqrlToSql.java
+++ b/sqrl-calcite/src/main/java/com/datasqrl/calcite/SqrlToSql.java
@@ -23,6 +23,7 @@ import com.datasqrl.canonicalizer.NamePath;
 import com.datasqrl.canonicalizer.ReservedName;
 import com.datasqrl.plan.hints.TopNHint.Type;
 import com.datasqrl.util.SqlNameUtil;
+import com.fasterxml.jackson.databind.jsontype.BasicPolymorphicTypeValidator.NameMatcher;
 import com.google.common.base.Preconditions;
 import com.google.common.collect.ArrayListMultimap;
 import com.google.common.collect.Multimap;
@@ -64,6 +65,7 @@ import org.apache.calcite.sql.SqlUnresolvedFunction;
 import org.apache.calcite.sql.fun.SqlStdOperatorTable;
 import org.apache.calcite.sql.parser.SqlParserPos;
 import org.apache.calcite.sql.util.SqlShuttle;
+import org.apache.calcite.sql.validate.SqlNameMatcher;
 
 public class SqrlToSql implements SqlRelationVisitor<Result, Context> {
   final QueryPlanner planner;
@@ -120,9 +122,11 @@ public class SqrlToSql implements SqlRelationVisitor<Result, Context> {
 
       List<String> columns = catalogReader.getTableFromPath(result.getCurrentPath()).get()
           .unwrap(ModifiableTable.class).getRowType().getFieldNames();
+      SqlNameMatcher nameMatcher = catalogReader.nameMatcher();
+
       List<SqlNode> newNodes = new ArrayList<>();
       for (String column : columns) {
-        if (!fieldNames.contains(column)) {
+        if (!containsFieldName(fieldNames, column, nameMatcher)) {
           newNodes.add(new SqlIdentifier(column, SqlParserPos.ZERO));
           fieldNames.add(column);
         }
@@ -167,6 +171,15 @@ public class SqrlToSql implements SqlRelationVisitor<Result, Context> {
 
     return new Result(select.build(), result.getCurrentPath(), List.of(), List.of(), Optional.empty(),
         result.params);
+  }
+
+  private boolean containsFieldName(Collection<String> fieldNames, String column, SqlNameMatcher nameMatcher) {
+    for (String fieldName : fieldNames) {
+      if (nameMatcher.matches(fieldName, column)) {
+        return true;
+      }
+    }
+    return false;
   }
 
   private void determineIsA(SqlSelect call, Context newContext, Result result) {

--- a/sqrl-testing/sqrl-integration-tests/src/test/resources/dagplanner/distinctTest.sqrl
+++ b/sqrl-testing/sqrl-integration-tests/src/test/resources/dagplanner/distinctTest.sqrl
@@ -10,4 +10,6 @@ CustomerByUpdatedAsc := DISTINCT Customer ON customerid ORDER BY lastUpdated DES
 CustomerByMultipleTime := DISTINCT Customer ON customerid, email ORDER BY timestamp DESC, lastUpdated ASC;
 CustomerByMultiple := DISTINCT Customer ON customerid, email ORDER BY lastUpdated DESC, timestamp ASC;
 
+CustomerVaryingCase := DISTINCT Customer ON customerID ORDER BY lastUpdated DESC;
+
 ExplicitDistinct := SELECT customerid, timestamp, name FROM (SELECT *, (ROW_NUMBER() OVER (PARTITION BY customerid ORDER BY timestamp DESC)) AS _rownum FROM Customer) WHERE (_rownum = 1);

--- a/sqrl-testing/sqrl-integration-tests/src/test/resources/snapshots/com/datasqrl/DAGPlannerTest/distinctTest.txt
+++ b/sqrl-testing/sqrl-integration-tests/src/test/resources/snapshots/com/datasqrl/DAGPlannerTest/distinctTest.txt
@@ -145,6 +145,25 @@ LogicalProject(customerid=[$0], email=[$1], name=[$2], lastUpdated=[$3], timesta
     LogicalProject(customerid=[$0], email=[$1], name=[$2], lastUpdated=[$3], timestamp=[$4], _rownum=[ROW_NUMBER() OVER (PARTITION BY $0 ORDER BY $3 DESC NULLS FIRST)])
       LogicalTableScan(table=[[customer_2]])
 
+=== CustomerVaryingCase
+ID:     customervaryingcase_1
+Type:   state
+Stage:  flink
+Inputs: customer_2
+Primary Key: customerID
+Timestamp  : timestamp
+Schema:
+ - customerID: BIGINT NOT NULL
+ - email: VARCHAR(2147483647) CHARACTER SET "UTF-16LE" NOT NULL
+ - name: VARCHAR(2147483647) CHARACTER SET "UTF-16LE" NOT NULL
+ - lastUpdated: BIGINT NOT NULL
+ - timestamp: TIMESTAMP_WITH_LOCAL_TIME_ZONE(3) NOT NULL
+Plan:
+LogicalProject(customerID=[$0], email=[$1], name=[$2], lastUpdated=[$3], timestamp=[$4])
+  LogicalFilter(condition=[=($5, 1)])
+    LogicalProject(customerID=[$0], email=[$1], name=[$2], lastUpdated=[$3], timestamp=[$4], _rownum=[ROW_NUMBER() OVER (PARTITION BY $0 ORDER BY $3 DESC NULLS FIRST)])
+      LogicalTableScan(table=[[customer_2]])
+
 === ExplicitDistinct
 ID:     explicitdistinct_1
 Type:   state
@@ -175,6 +194,7 @@ LogicalProject(customerid=[$0], timestamp=[$4], name=[$2])
     "CREATE TEMPORARY TABLE `customerbytimeasc_1` (\n  `customerid` BIGINT NOT NULL,\n  `email` VARCHAR(2147483647) CHARACTER SET `UTF-16LE` NOT NULL,\n  `name` VARCHAR(2147483647) CHARACTER SET `UTF-16LE` NOT NULL,\n  `lastUpdated` BIGINT NOT NULL,\n  `timestamp` TIMESTAMP(3) WITH LOCAL TIME ZONE NOT NULL,\n  PRIMARY KEY (`customerid`) NOT ENFORCED\n) WITH (\n  'password' = '${JDBC_PASSWORD}',\n  'connector' = 'jdbc-sqrl',\n  'driver' = 'org.postgresql.Driver',\n  'table-name' = 'customerbytimeasc_1',\n  'url' = '${JDBC_URL}',\n  'username' = '${JDBC_USERNAME}'\n);",
     "CREATE TEMPORARY TABLE `customerbyupdated_1` (\n  `customerid` BIGINT NOT NULL,\n  `email` VARCHAR(2147483647) CHARACTER SET `UTF-16LE` NOT NULL,\n  `name` VARCHAR(2147483647) CHARACTER SET `UTF-16LE` NOT NULL,\n  `lastUpdated` BIGINT NOT NULL,\n  `timestamp` TIMESTAMP(3) WITH LOCAL TIME ZONE NOT NULL,\n  PRIMARY KEY (`customerid`) NOT ENFORCED\n) WITH (\n  'password' = '${JDBC_PASSWORD}',\n  'connector' = 'jdbc-sqrl',\n  'driver' = 'org.postgresql.Driver',\n  'table-name' = 'customerbyupdated_1',\n  'url' = '${JDBC_URL}',\n  'username' = '${JDBC_USERNAME}'\n);",
     "CREATE TEMPORARY TABLE `customerbyupdatedasc_1` (\n  `customerid` BIGINT NOT NULL,\n  `email` VARCHAR(2147483647) CHARACTER SET `UTF-16LE` NOT NULL,\n  `name` VARCHAR(2147483647) CHARACTER SET `UTF-16LE` NOT NULL,\n  `lastUpdated` BIGINT NOT NULL,\n  `timestamp` TIMESTAMP(3) WITH LOCAL TIME ZONE NOT NULL,\n  PRIMARY KEY (`customerid`) NOT ENFORCED\n) WITH (\n  'password' = '${JDBC_PASSWORD}',\n  'connector' = 'jdbc-sqrl',\n  'driver' = 'org.postgresql.Driver',\n  'table-name' = 'customerbyupdatedasc_1',\n  'url' = '${JDBC_URL}',\n  'username' = '${JDBC_USERNAME}'\n);",
+    "CREATE TEMPORARY TABLE `customervaryingcase_1` (\n  `customerID` BIGINT NOT NULL,\n  `email` VARCHAR(2147483647) CHARACTER SET `UTF-16LE` NOT NULL,\n  `name` VARCHAR(2147483647) CHARACTER SET `UTF-16LE` NOT NULL,\n  `lastUpdated` BIGINT NOT NULL,\n  `timestamp` TIMESTAMP(3) WITH LOCAL TIME ZONE NOT NULL,\n  PRIMARY KEY (`customerID`) NOT ENFORCED\n) WITH (\n  'password' = '${JDBC_PASSWORD}',\n  'connector' = 'jdbc-sqrl',\n  'driver' = 'org.postgresql.Driver',\n  'table-name' = 'customervaryingcase_1',\n  'url' = '${JDBC_URL}',\n  'username' = '${JDBC_USERNAME}'\n);",
     "CREATE TEMPORARY TABLE `explicitdistinct_1` (\n  `customerid` BIGINT NOT NULL,\n  `timestamp` TIMESTAMP(3) WITH LOCAL TIME ZONE NOT NULL,\n  `name` VARCHAR(2147483647) CHARACTER SET `UTF-16LE` NOT NULL,\n  PRIMARY KEY (`customerid`) NOT ENFORCED\n) WITH (\n  'password' = '${JDBC_PASSWORD}',\n  'connector' = 'jdbc-sqrl',\n  'driver' = 'org.postgresql.Driver',\n  'table-name' = 'explicitdistinct_1',\n  'url' = '${JDBC_URL}',\n  'username' = '${JDBC_USERNAME}'\n);",
     "CREATE VIEW `table$1`\nAS\nSELECT *\nFROM `customer_1`;",
     "CREATE VIEW `table$2`\nAS\nSELECT `customerid`, `email`, `name`, `lastUpdated`, `timestamp`\nFROM (SELECT `customerid`, `email`, `name`, `lastUpdated`, `timestamp`, ROW_NUMBER() OVER (PARTITION BY `customerid`, `email` ORDER BY `lastUpdated` DESC, `timestamp`) AS `_rownum`\n  FROM `customer_1`) AS `t`\nWHERE `_rownum` = 1;",
@@ -184,8 +204,9 @@ LogicalProject(customerid=[$0], timestamp=[$4], name=[$2])
     "CREATE VIEW `table$6`\nAS\nSELECT `customerid`, `email`, `name`, `lastUpdated`, `timestamp`\nFROM (SELECT `customerid`, `email`, `name`, `lastUpdated`, `timestamp`, ROW_NUMBER() OVER (PARTITION BY `customerid` ORDER BY `timestamp`) AS `_rownum`\n  FROM `customer_1`) AS `t`\nWHERE `_rownum` = 1;",
     "CREATE VIEW `table$7`\nAS\nSELECT `customerid`, `email`, `name`, `lastUpdated`, `timestamp`\nFROM (SELECT `customerid`, `email`, `name`, `lastUpdated`, `timestamp`, ROW_NUMBER() OVER (PARTITION BY `customerid` ORDER BY `lastUpdated` DESC) AS `_rownum`\n  FROM `customer_1`) AS `t`\nWHERE `_rownum` = 1;",
     "CREATE VIEW `table$8`\nAS\nSELECT `customerid`, `email`, `name`, `lastUpdated`, `timestamp`\nFROM (SELECT `customerid`, `email`, `name`, `lastUpdated`, `timestamp`, ROW_NUMBER() OVER (PARTITION BY `customerid` ORDER BY `lastUpdated` DESC) AS `_rownum`\n  FROM `customer_1`) AS `t`\nWHERE `_rownum` = 1;",
-    "CREATE VIEW `table$9`\nAS\nSELECT `customerid`, `timestamp`, `name`\nFROM (SELECT `customerid`, `email`, `name`, `lastUpdated`, `timestamp`, ROW_NUMBER() OVER (PARTITION BY `customerid` ORDER BY `timestamp` DESC) AS `_rownum`\n  FROM `customer_1`) AS `t`\nWHERE `_rownum` = 1;",
-    "EXECUTE STATEMENT SET BEGIN\nINSERT INTO `customer_2`\n(SELECT *\n FROM `table$1`)\n;\nINSERT INTO `customerbymultiple_1`\n (SELECT *\n  FROM `table$2`)\n ;\n INSERT INTO `customerbymultipletime_1`\n  (SELECT *\n   FROM `table$3`)\n  ;\n  INSERT INTO `customerbytime2_1`\n   (SELECT *\n    FROM `table$4`)\n   ;\n   INSERT INTO `customerbytime_1`\n    (SELECT *\n     FROM `table$5`)\n    ;\n    INSERT INTO `customerbytimeasc_1`\n     (SELECT *\n      FROM `table$6`)\n     ;\n     INSERT INTO `customerbyupdated_1`\n      (SELECT *\n       FROM `table$7`)\n      ;\n      INSERT INTO `customerbyupdatedasc_1`\n       (SELECT *\n        FROM `table$8`)\n       ;\n       INSERT INTO `explicitdistinct_1`\n        (SELECT *\n         FROM `table$9`)\n        ;\n        END;"
+    "CREATE VIEW `table$9`\nAS\nSELECT `customerID`, `email`, `name`, `lastUpdated`, `timestamp`\nFROM (SELECT `customerid` AS `customerID`, `email`, `name`, `lastUpdated`, `timestamp`, ROW_NUMBER() OVER (PARTITION BY `customerid` ORDER BY `lastUpdated` DESC) AS `_rownum`\n  FROM `customer_1`) AS `t`\nWHERE `_rownum` = 1;",
+    "CREATE VIEW `table$10`\nAS\nSELECT `customerid`, `timestamp`, `name`\nFROM (SELECT `customerid`, `email`, `name`, `lastUpdated`, `timestamp`, ROW_NUMBER() OVER (PARTITION BY `customerid` ORDER BY `timestamp` DESC) AS `_rownum`\n  FROM `customer_1`) AS `t`\nWHERE `_rownum` = 1;",
+    "EXECUTE STATEMENT SET BEGIN\nINSERT INTO `customer_2`\n(SELECT *\n FROM `table$1`)\n;\nINSERT INTO `customerbymultiple_1`\n (SELECT *\n  FROM `table$2`)\n ;\n INSERT INTO `customerbymultipletime_1`\n  (SELECT *\n   FROM `table$3`)\n  ;\n  INSERT INTO `customerbytime2_1`\n   (SELECT *\n    FROM `table$4`)\n   ;\n   INSERT INTO `customerbytime_1`\n    (SELECT *\n     FROM `table$5`)\n    ;\n    INSERT INTO `customerbytimeasc_1`\n     (SELECT *\n      FROM `table$6`)\n     ;\n     INSERT INTO `customerbyupdated_1`\n      (SELECT *\n       FROM `table$7`)\n      ;\n      INSERT INTO `customerbyupdatedasc_1`\n       (SELECT *\n        FROM `table$8`)\n       ;\n       INSERT INTO `customervaryingcase_1`\n        (SELECT *\n         FROM `table$9`)\n        ;\n        INSERT INTO `explicitdistinct_1`\n         (SELECT *\n          FROM `table$10`)\n         ;\n         END;"
   ],
   "connectors" : [
     "jdbc-sqrl",
@@ -316,6 +337,20 @@ LogicalProject(customerid=[$0], timestamp=[$4], name=[$2])
         "\"customerid\""
       ],
       "sql" : "CREATE TABLE IF NOT EXISTS customerbyupdatedasc_1 (\"customerid\" BIGINT NOT NULL,\"email\" TEXT NOT NULL,\"name\" TEXT NOT NULL,\"lastUpdated\" BIGINT NOT NULL,\"timestamp\" TIMESTAMP WITH TIME ZONE NOT NULL , PRIMARY KEY (\"customerid\"));"
+    },
+    {
+      "name" : "customervaryingcase_1",
+      "columns" : [
+        "\"customerID\" BIGINT NOT NULL",
+        "\"email\" TEXT NOT NULL",
+        "\"name\" TEXT NOT NULL",
+        "\"lastUpdated\" BIGINT NOT NULL",
+        "\"timestamp\" TIMESTAMP WITH TIME ZONE NOT NULL"
+      ],
+      "primaryKeys" : [
+        "\"customerID\""
+      ],
+      "sql" : "CREATE TABLE IF NOT EXISTS customervaryingcase_1 (\"customerID\" BIGINT NOT NULL,\"email\" TEXT NOT NULL,\"name\" TEXT NOT NULL,\"lastUpdated\" BIGINT NOT NULL,\"timestamp\" TIMESTAMP WITH TIME ZONE NOT NULL , PRIMARY KEY (\"customerID\"));"
     },
     {
       "name" : "explicitdistinct_1",
@@ -632,6 +667,26 @@ LogicalProject(customerid=[$0], timestamp=[$4], name=[$2])
               {
                 "type" : "variable",
                 "type" : "variable",
+                "path" : "limit"
+              },
+              {
+                "type" : "variable",
+                "type" : "variable",
+                "path" : "offset"
+              }
+            ],
+            "query" : {
+              "type" : "PagedJdbcQuery",
+              "type" : "PagedJdbcQuery",
+              "sql" : "SELECT *\nFROM \"customerbymultipletime_1\"\nORDER BY \"customerid\", \"email\"",
+              "parameters" : [ ]
+            }
+          },
+          {
+            "arguments" : [
+              {
+                "type" : "variable",
+                "type" : "variable",
                 "path" : "customerid"
               },
               {
@@ -697,26 +752,6 @@ LogicalProject(customerid=[$0], timestamp=[$4], name=[$2])
                   "path" : "customerid"
                 }
               ]
-            }
-          },
-          {
-            "arguments" : [
-              {
-                "type" : "variable",
-                "type" : "variable",
-                "path" : "limit"
-              },
-              {
-                "type" : "variable",
-                "type" : "variable",
-                "path" : "offset"
-              }
-            ],
-            "query" : {
-              "type" : "PagedJdbcQuery",
-              "type" : "PagedJdbcQuery",
-              "sql" : "SELECT *\nFROM \"customerbymultipletime_1\"\nORDER BY \"customerid\", \"email\"",
-              "parameters" : [ ]
             }
           },
           {
@@ -1045,6 +1080,64 @@ LogicalProject(customerid=[$0], timestamp=[$4], name=[$2])
       {
         "type" : "args",
         "parentType" : "Query",
+        "fieldName" : "CustomerVaryingCase",
+        "matchs" : [
+          {
+            "arguments" : [
+              {
+                "type" : "variable",
+                "type" : "variable",
+                "path" : "customerID"
+              },
+              {
+                "type" : "variable",
+                "type" : "variable",
+                "path" : "limit"
+              },
+              {
+                "type" : "variable",
+                "type" : "variable",
+                "path" : "offset"
+              }
+            ],
+            "query" : {
+              "type" : "PagedJdbcQuery",
+              "type" : "PagedJdbcQuery",
+              "sql" : "SELECT *\nFROM \"customervaryingcase_1\"\nWHERE \"customerID\" = $1",
+              "parameters" : [
+                {
+                  "type" : "arg",
+                  "type" : "arg",
+                  "path" : "customerID"
+                }
+              ]
+            }
+          },
+          {
+            "arguments" : [
+              {
+                "type" : "variable",
+                "type" : "variable",
+                "path" : "limit"
+              },
+              {
+                "type" : "variable",
+                "type" : "variable",
+                "path" : "offset"
+              }
+            ],
+            "query" : {
+              "type" : "PagedJdbcQuery",
+              "type" : "PagedJdbcQuery",
+              "sql" : "SELECT *\nFROM \"customervaryingcase_1\"\nORDER BY \"customerID\"",
+              "parameters" : [ ]
+            }
+          }
+        ]
+      },
+      {
+        "type" : "args",
+        "parentType" : "Query",
         "fieldName" : "ExplicitDistinct",
         "matchs" : [
           {
@@ -1106,7 +1199,7 @@ LogicalProject(customerid=[$0], timestamp=[$4], name=[$2])
     "schema" : {
       "type" : "string",
       "type" : "string",
-      "schema" : "type Customer {\n  customerid: Float!\n  email: String!\n  name: String!\n  lastUpdated: Float!\n  timestamp: DateTime!\n}\n\ntype CustomerByMultiple {\n  customerid: Float!\n  email: String!\n  name: String!\n  lastUpdated: Float!\n  timestamp: DateTime!\n}\n\ntype CustomerByMultipleTime {\n  customerid: Float!\n  email: String!\n  name: String!\n  lastUpdated: Float!\n  timestamp: DateTime!\n}\n\ntype CustomerByTime {\n  customerid: Float!\n  email: String!\n  name: String!\n  lastUpdated: Float!\n  timestamp: DateTime!\n}\n\ntype CustomerByTime2 {\n  customerid: Float!\n  email: String!\n  name: String!\n  lastUpdated: Float!\n  timestamp: DateTime!\n}\n\ntype CustomerByTimeAsc {\n  customerid: Float!\n  email: String!\n  name: String!\n  lastUpdated: Float!\n  timestamp: DateTime!\n}\n\ntype CustomerByUpdated {\n  customerid: Float!\n  email: String!\n  name: String!\n  lastUpdated: Float!\n  timestamp: DateTime!\n}\n\ntype CustomerByUpdatedAsc {\n  customerid: Float!\n  email: String!\n  name: String!\n  lastUpdated: Float!\n  timestamp: DateTime!\n}\n\n\"An RFC-3339 compliant DateTime Scalar\"\nscalar DateTime\n\ntype ExplicitDistinct {\n  customerid: Float!\n  timestamp: DateTime!\n  name: String!\n}\n\ntype Query {\n  Customer(customerid: Float, lastUpdated: Float, limit: Int = 10, offset: Int = 0): [Customer!]\n  CustomerByMultiple(customerid: Float, email: String, limit: Int = 10, offset: Int = 0): [CustomerByMultiple!]\n  CustomerByMultipleTime(customerid: Float, email: String, limit: Int = 10, offset: Int = 0): [CustomerByMultipleTime!]\n  CustomerByTime(customerid: Float, limit: Int = 10, offset: Int = 0): [CustomerByTime!]\n  CustomerByTime2(customerid: Float, limit: Int = 10, offset: Int = 0): [CustomerByTime2!]\n  CustomerByTimeAsc(customerid: Float, limit: Int = 10, offset: Int = 0): [CustomerByTimeAsc!]\n  CustomerByUpdated(customerid: Float, limit: Int = 10, offset: Int = 0): [CustomerByUpdated!]\n  CustomerByUpdatedAsc(customerid: Float, limit: Int = 10, offset: Int = 0): [CustomerByUpdatedAsc!]\n  ExplicitDistinct(customerid: Float, limit: Int = 10, offset: Int = 0): [ExplicitDistinct!]\n}\n"
+      "schema" : "type Customer {\n  customerid: Float!\n  email: String!\n  name: String!\n  lastUpdated: Float!\n  timestamp: DateTime!\n}\n\ntype CustomerByMultiple {\n  customerid: Float!\n  email: String!\n  name: String!\n  lastUpdated: Float!\n  timestamp: DateTime!\n}\n\ntype CustomerByMultipleTime {\n  customerid: Float!\n  email: String!\n  name: String!\n  lastUpdated: Float!\n  timestamp: DateTime!\n}\n\ntype CustomerByTime {\n  customerid: Float!\n  email: String!\n  name: String!\n  lastUpdated: Float!\n  timestamp: DateTime!\n}\n\ntype CustomerByTime2 {\n  customerid: Float!\n  email: String!\n  name: String!\n  lastUpdated: Float!\n  timestamp: DateTime!\n}\n\ntype CustomerByTimeAsc {\n  customerid: Float!\n  email: String!\n  name: String!\n  lastUpdated: Float!\n  timestamp: DateTime!\n}\n\ntype CustomerByUpdated {\n  customerid: Float!\n  email: String!\n  name: String!\n  lastUpdated: Float!\n  timestamp: DateTime!\n}\n\ntype CustomerByUpdatedAsc {\n  customerid: Float!\n  email: String!\n  name: String!\n  lastUpdated: Float!\n  timestamp: DateTime!\n}\n\ntype CustomerVaryingCase {\n  customerID: Float!\n  email: String!\n  name: String!\n  lastUpdated: Float!\n  timestamp: DateTime!\n}\n\n\"An RFC-3339 compliant DateTime Scalar\"\nscalar DateTime\n\ntype ExplicitDistinct {\n  customerid: Float!\n  timestamp: DateTime!\n  name: String!\n}\n\ntype Query {\n  Customer(customerid: Float, lastUpdated: Float, limit: Int = 10, offset: Int = 0): [Customer!]\n  CustomerByMultiple(customerid: Float, email: String, limit: Int = 10, offset: Int = 0): [CustomerByMultiple!]\n  CustomerByMultipleTime(customerid: Float, email: String, limit: Int = 10, offset: Int = 0): [CustomerByMultipleTime!]\n  CustomerByTime(customerid: Float, limit: Int = 10, offset: Int = 0): [CustomerByTime!]\n  CustomerByTime2(customerid: Float, limit: Int = 10, offset: Int = 0): [CustomerByTime2!]\n  CustomerByTimeAsc(customerid: Float, limit: Int = 10, offset: Int = 0): [CustomerByTimeAsc!]\n  CustomerByUpdated(customerid: Float, limit: Int = 10, offset: Int = 0): [CustomerByUpdated!]\n  CustomerByUpdatedAsc(customerid: Float, limit: Int = 10, offset: Int = 0): [CustomerByUpdatedAsc!]\n  CustomerVaryingCase(customerID: Float, limit: Int = 10, offset: Int = 0): [CustomerVaryingCase!]\n  ExplicitDistinct(customerid: Float, limit: Int = 10, offset: Int = 0): [ExplicitDistinct!]\n}\n"
     }
   }
 }

